tests/conftest.py

```diff
@@ -26,6 +26,10 @@ def pytest_addoption(parser):
         help="The LIN interface to use with the tests")
 
 
+def pytest_configure(config):
+    config.addinivalue_line("markers", "integration: Integration test.")
```

Comment:
In general, we usually place this in the config file under the [pytest] section. For example,
```ini
[pytest]
addopts = --cov nixnet --cov nixnet_examples --cov-report term --cov-report xml --verbose --doctest-modules --ignore=setup.py
markers =
    integration: Integration test.
```
I would only use pytest_configure if the settings cannot be placed in a config file.

Reply:
Thanks @tkrebes for the suggestion, I've moved the marker configuration to the tox INI files instead of conftest.py in the latest commit.